### PR TITLE
Create jhardyjhardy.json

### LINKED ISSUE #4008: Joining
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by jhardyjhardy**
Added my pin
@githubteacher 
closes #4008

### CODE REVIEW

**Comment by zafarhussaini** [comment]
You just need the dashes.

File: _pins/jhardyjhardy.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: jhardyjhardy
```
Comment (borzumate) [inline _pins/jhardyjhardy.json]: Need three dashes above this line; also at the end of the file

Comment (hectorsector) [inline _pins/jhardyjhardy.json]: @jhardyjhardy 
Great work so far! As @borzumate suggested, you'll need to make sure that your code matches exactly the template in the class, including `---` before and after the code:
```
---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---
```

Comment (wdaali) [inline _pins/jhardyjhardy.json]: @jhardyjhardy  Great job! :smile:

**Comment by sSoKoSs** [comment]
Good job :+1: :grinning:

**Comment by brianamarie** [comment]
Hi @jhardyjhardy! I'm going to go ahead and make the changes now.

**Comment by brianamarie** [comment]
Woohoo, @jhardyjhardy! I'm going to merge now. Nice work!

**Pull request merged**